cmd/ethrex/Cargo.toml

```diff
@@ -52,6 +51,7 @@ reqwest.workspace = true
 thiserror.workspace = true
 itertools = "0.14.0"
 url.workspace = true
+semver = "1.0.27"
```

Comment:
We're not following semver right now, so this is probably not the right tool to use.
I would just use a simple inequality check against `latest`.

Reply:
In that case, I would still not use a crate that assumes semver in a package that explicitly opts out of semver. A simple function that splits and parses integers and checks whether any is less than the latest version's one should suffice.

Reply:
Added manual version comparison: [f637f4e](https://github.com/lambdaclass/ethrex/pull/5322/commits/f637f4e8f14f6a3b389a8da936ce75d8d4ecae3a)